src/common/hist_util.cu

```diff
@@ -22,94 +22,14 @@
 #include "../data/device_adapter.cuh"
 #include "device_helpers.cuh"
```

Comment:
Is there any point in keeping this file now?

Reply:
Em, let me remove it.  Thanks!